Products/Archetypes/skins/archetypes/edit_macros.pt

```diff
@@ -80,11 +80,12 @@
             method="post"
             enctype="multipart/form-data"
             class="enableUnloadProtection enableAutoFocus atBaseEditForm"
+            data-pat-autotoc="levels: legend; section: fieldset; className: autotabs"
             action=""
             id=""
             tal:attributes="action python:context.absolute_url()+'/'+template.id;
                             id string:${portal_type}-base-edit;
-                            class python:test(path('allow_tabbing|nothing'), 'enableUnloadProtection enableAutoFocus enableFormTabbing enableUnlockProtection', 'enableUnloadProtection enableAutoFocus enableUnlockProtection');">
+                            class python:test(path('allow_tabbing|nothing'), 'enableUnloadProtection enableAutoFocus enableFormTabbing enableUnlockProtection pat-autotoc', 'enableUnloadProtection enableAutoFocus enableUnlockProtection');">
```

Comment:
I'm -1 on changing this, for:

1. Archetypes is old; not much return on investment of time.
2. There are 26 instances of this idiom in templates in skins/archetypes and subdirectories.
3. Making this change to Py26 conditional expressions could have been done ages ago, but has not (yet).
4. We risk (by mere possibility of a typo) breaking more than we fix for syntactic sugar?  I'm not going to be confident that all these 26 uses of test() in templates is going to have adequate test coverage.